app/models/product.rb

```diff
@@ -34,4 +36,12 @@ def ensure_not_referenced_by_any_line_item
         throw :abort
       end
     end
+
+    def give_default_title
+      self.title='abc'
+    end
+
+    def set_default_discount_price
+      self.discount_price = self.price
```

Comment:
will this work with our validation where price should be greater than discount price?